SX1276/SX1276_LoRaRadio.cpp

```diff
@@ -667,22 +667,21 @@ uint32_t SX1276_LoRaRadio::time_on_air(radio_modems_t modem, uint8_t pkt_len)
 
     switch (modem) {
         case MODEM_FSK:
-            airTime =
-                    rint((8 * (_rf_settings.fsk.preamble_len
+            uint64_t tmp = 1000 * 8 * (_rf_settings.fsk.preamble_len
                                     + ((read_register( REG_SYNCCONFIG)
                                             & ~RF_SYNCCONFIG_SYNCSIZE_MASK) + 1)
                                     + ((_rf_settings.fsk.fix_len == 0x01) ?
-                                            0.0f : 1.0f)
+                                            0 : 1)
                                     + (((read_register( REG_PACKETCONFIG1)
                                             & ~RF_PACKETCONFIG1_ADDRSFILTERING_MASK)
-                                            != 0x00) ? 1.0f : 0) + pkt_len
+                                            != 0x00) ? 1 : 0) + pkt_len
                                     + ((_rf_settings.fsk.crc_on == 0x01) ?
-                                            2.0 : 0))
-                            / _rf_settings.fsk.datarate) * 1000);
+                                            2 : 0));
+            airTime = (tmp / _rf_settings.fsk.datarate) + (((tmp % _rf_settings.fsk.datarate) > 0) ? 1 : 0);
 
             break;
         case MODEM_LORA:
-            float bw = 0.0f;
+            int bw = 0;
```

Comment:
Is there a reason for not using `int32_t` or `int64_t`?

Reply:
No reason, will use uint32_t for consistency.